doc/classes/AnimationPlayer.xml

```diff
@@ -145,6 +146,7 @@
 			<description>
 				Plays the animation with key [code]name[/code]. Custom speed and blend times can be set. If [code]custom_speed[/code] is negative and [code]from_end[/code] is [code]true[/code], the animation will play backwards.
 				If the animation has been paused by [method stop], it will be resumed. Calling [method play] without arguments will also resume the animation.
+				[b]Note:[/b] The animation will be updated the next time the AnimationPlayer is processed. If other variables are updated at the same time this is called, they may be updated too early. To perform the update immediately, call: [code]advance(0)[/code]
```

Comment:
This is more natural-sounding to me:

```suggestion
				[b]Note:[/b] The animation will be updated the next time the AnimationPlayer is processed. If other variables are updated at the same time this is called, they may be updated too early. To perform the update immediately, call [code]advance(0)[/code].
```

Reply:
I always worry that `codehere`. will be read as `codehere.`, but committing suggestion.